fast64_internal/sm64/sm64_level_writer.py

```diff
@@ -124,12 +127,7 @@ def __init__(self, masks, originalData):
         self.originalData = originalData
 
     def to_c(self):
-        result = ""
-        # result += 'u8 sZoomOutAreaMasks[] = {\n'
-        result += "\n"
```

Comment:
Please add the new line back in the write function for zoom out masks, doesn´t break anything, looks better and there is no reason to cause unnecessary changes like that
![image](https://github.com/Fast-64/fast64/assets/87947656/831d21e1-23a5-4177-a8ec-ae316cb8b8e4)